statsmodels/emplike/tests/test_regression.py

```diff
@@ -13,11 +13,12 @@ class GenRes(object):
     Loads data and creates class instance ot be tested
 
     """
-    def __init__(self):
+    @classmethod
+    def setup_class(cls):
```

Comment:
`__init__` is mostly bad practice.  But in this case pytest doesn't run and tests in classes with `__init__`.  Memory use was very mild on my machine ~175MB/process max.